core/ui/prime/components/src/lib/input-base/input-base.component.ts

```diff
@@ -1,85 +1,81 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Why has this file so many changes? Spaces to tab?

Reply:
Also structured the code a bit and do some formatting